Sources/AppStoreConnectCLI/Services/Operations/GetAppsOperation.swift

```diff
@@ -0,0 +1,56 @@
+// Copyright 2020 Itty Bitty Apps Pty Ltd
+
+import AppStoreConnect_Swift_SDK
+import Combine
+import Foundation
+
+struct GetAppsOperation: APIOperation {
+    struct GetAppsDependencies {
+        let apps: (APIEndpoint<AppsResponse>) -> Future<AppsResponse, Error>
+    }
+
+    enum GetAppIdsError: LocalizedError {
+        case couldntFindAnyAppsMatching(bundleIds: [String])
+        case appsDoNotExist(bundleIds: [String])
+
+        var errorDescription: String? {
+            switch self {
+            case .couldntFindAnyAppsMatching(let bundleIds):
+                return "No apps were found matching \(bundleIds)"
+            case .appsDoNotExist(let bundleIds):
+                return "Specified apps were non found / do not exist: \(bundleIds)"
+            }
+        }
+    }
+
+    private let options: GetAppsOptions
+
+    init(options: GetAppsOptions) {
+        self.options = options
+    }
+
+    func execute(
+        with dependencies: GetAppsDependencies
+    ) -> AnyPublisher<[AppStoreConnect_Swift_SDK.App], Error> {
```

Comment:
We have our own App Model as well, and I understand sometimes we need the id from the SDK model to fetch further information. I was wondering how do we solve the incompatible? eg. Can we use our `App` model here: `AnyPublisher<[Apps], Error>`?

Reply:
I think we can move to our own App model, yep

Reply:
> I think we can move to our own App model, yep

But I think you need the APP internal resource id when fetching stuff. Our App Model doesn't have that. What are your thoughts on this?

Reply:
I think that gives us more of a reason to separate our models and networking into a separate module. That way we can have the internal id be `internal` access level and the rest of it be `public`